src/tools/genpolicy/genpolicy-settings.json

```diff
@@ -324,6 +324,24 @@
             ],
             "allow_env_regex_map": {}
         },
+        "UpdateInterfaceRequest": {
+            "interface": {
+                "device": "eth0",
+                "name": "eth0",
+                "IPAddresses": [
+                    {
+                        "family": 0,
+                        "address": "10.244.0.14",
```

Comment:
Our main goal here is to guard Guest's loopback interface, if needed. I don't think it's worth trying to hard-code non-loopback IP addresses.

Reply:
Okay, I removed the policy checks for IPAddresses